data/hamster.metainfo.xml

```diff
@@ -27,7 +27,7 @@
     </ul>
   </description>
 
-  <launchable type="desktop-id">hamster.desktop</launchable>
+  <launchable type="desktop-id">org.gnome.Hamster.GtkApp.desktop</launchable>
```

Comment:
I was thinking of flatpaking Hamster, and renaming the desktop file (and other resources as required by Flatpak) is something I was going to do.

So a big :+1: for this.